code/modules/modular_computers/file_system/programs/generic/file_browser.dm

```diff
@@ -55,7 +55,8 @@
 					"name" = F.filename,
 					"type" = F.filetype,
 					"size" = F.size,
-					"undeletable" = F.undeletable
+					"undeletable" = F.undeletable,
+					"encrypted" = !!F.password
```

Comment:
Why is there a `!!` here (and down below)?

Reply:
To change it to a boolean without sending the password to the client from what I know

Reply:
Yee, byond shitcode.